class-tgm-plugin-activation.php

```diff
@@ -249,7 +249,7 @@ public function init() {
 			// After this point, the plugins should be registered and the configuration set.
 
 			// Proceed only if we have plugins to handle.
-			if ( $this->plugins ) {
+			if ( ! empty( $this->plugins ) ) {
 				$sorted = array();
```

Comment:
There's no technical reason for `is_array()` being first, and according to the bottom of http://www.phpbench.com/, `is_array()` is relatively expensive to check compared to seeing if it's some type that is empty. 

So: "Does this variable have a non-nil value, and is it an array?" may be quicker to abort for bad values compared to "Is this variable an array, and does it have a non-nil value?"

Of course, this is a micro-optimisation, so I'm happy with the already committed order.